server/service/service.go

```diff
@@ -81,8 +89,41 @@ func NewService(
 		failingPolicySet: failingPolicySet,
 		authz:            authorizer,
 	}
-	svc = validationMiddleware{svc, ds, sso}
-	return svc, nil
+
+	// Try setting a first seed
+	_ = svc.updateJitterSeedRand()
+	go svc.updateJitterSeed(ctx)
+
+	return validationMiddleware{svc, ds, sso}, nil
+}
+
+func (s *Service) updateJitterSeedRand() error {
+	nBig, err := rand.Int(rand.Reader, big.NewInt(math.MaxInt))
+	if err != nil {
+		return err
+	}
+	n := nBig.Int64()
+	atomic.StoreInt64(&s.jitterSeed, n)
+	return nil
+}
+
+func (s *Service) updateJitterSeed(ctx context.Context) {
+	for {
+		select {
+		case <-time.Tick(1 * time.Hour):
```

Comment:
Can you remind me why `1h`? (maybe we can have one seed per group, one for `labels`, one for `policies` and one for `details`; they could differ depending on the deployment).

Reply:
Just because that's the interval that was discussed in the call. I would prefer to see if we can make this approach work before making it more complex (unless make it simpler means it doesn't work heh)